src/pixi/IgePixiAnimation.js

```diff
@@ -88,7 +88,13 @@ var IgePixiAnimation = IgeClass.extend({
 	},
 	select: function (frames, fps = 15, loopCount, cellSheetAnimId, animName) {
 		var self = this;
-		var entity = ige.pixi.trackEntityById[self._entity.entityId];
+		if (
+			ige.entitiesToRender.trackEntityById[self._entity.entityId] &&
+			ige.entitiesToRender.trackEntityById[self._entity.entityId]._pixiContainer
+		) {
+			var entity = ige.entitiesToRender.trackEntityById[self._entity.entityId]._pixiContainer;
```

Comment:
nice. surprised i missed this earlier. the last updates on this code would have just been formatting, but since you point it out, we should probably just fix it.

Reply:
i proposed a solution with a change, but i will not resolve this conversation.